EmptyChronicle/Controller/StatesController.cs

```diff
@@ -0,0 +1,31 @@
+using EmptyChronicle.Application.States;
+using EmptyChronicle.Utility;
+using Microsoft.AspNetCore.Mvc;
+
+namespace EmptyChronicle.Controller;
+
+[Route("/api/states")]
+[ApiController]
+public class StatesController : ControllerBase
+{
+    private readonly StatesApplication StatesApplication;
+
+    public StatesController(StatesApplication statesApplication)
+    {
+        StatesApplication = statesApplication;
+    }
+
+    [HttpGet("{address}")]
+    public ActionResult GetStatesByAddress(string address, [FromQuery(Name = "account")] string? accountAddress)
+    {
+        var state = StatesApplication.GetStateByAddress(address, accountAddress);
```

Comment:
It should return `BadRequest()` when the `StatesApplication.GetStateByAddress` throws `ArgumentException` by [`Libplanet.Address.#ctor(string)`](https://docs.libplanet.io/4.0.6/api/Libplanet.Crypto.Address.html#Libplanet_Crypto_Address__ctor_System_String_).